README.md

```diff
@@ -0,0 +1,58 @@
+# nagios-plugins-bdii
+
+This nagios probe is used to monitor BDII.
+It provides check_bdii_entries support for Nagios.
+
+BDII documentation is available here: https://gridinfo-documentation.readthedocs.io/
+
+## Building packages
+
+A Makefile allowing to build source tarball and packages is provided.
+
+### Building a RPM
+
+The required build dependencies are:
+- rpm-build
+- make
+- rsync
+
+```sh
+# Checkout tag to be packaged
+git clone https://github.com/EGI-Foundation/nagios-plugins-bdii.git
+cd nagios-plugins-bdii
+git checkout X.X.X
+# Building in a container
+docker run --rm -v $(pwd):/source -it centos:7
+yum install -y rpm-build make rsync gcc openldap-devel
+cd /source && make rpm
+```
+
+The RPM will be available into the `build/RPMS` directory.
+
+### Building a deb
+
+**The alien-based deb pacakge creation is currently broken (rpm task doesn't work on Ubuntu/Debian)**
```

Comment:
It would be good for future us to add the name of the distro on which it currently doesn't work

Reply:
It's on all Ubuntu/Debian that it won't work as it is in fact.